kong/clustering/data_plane.lua

```diff
@@ -60,7 +61,7 @@ function _M:init_worker(plugins_list)
 
   self.plugins_list = plugins_list
 
-  if ngx.worker.id() == 0 then
+  if process.type() == "privileged agent" then
```

Comment:
Could we define a help function like `is_dp_worker_process` in `clusteringutils.lua`,?
then we will remove this duplicated code `process.type() == "privileged agent"`.